gulpfile.js

```diff
@@ -9,11 +9,6 @@ var buildPath = './demo/build/';
 
 gulp.task('auto-reload', loadLmnTask('auto-reload'));
 
-gulp.task('html', loadLmnTask('html', {
-  langBase: 'component.flick',
-  imagePath: '../../src/imgs/'
-}));
-
```

Comment:
update to `lmn-gulp-tasks@1.3.5`, there was a bug in `rubbish-erb-parser` in an older version and I forgot to update lmn-gulp-tasks.